src/Network/Email/Email.php

```diff
@@ -1241,6 +1241,16 @@ public static function configTransport($key, $config = null)
     }
 
     /**
+     * Returns an array containing the named transport configurations
+     *
+     * @return array Array of configurations.
+     */
+    public static function configuredTransport()
```

Comment:
I anticipated that argument, but semantically it can go both ways: If `configTransport()` "typically" takes a list, then shouldn't it be named with an 's' too? Conversely, if `configuredTransport()` is "typically" going to return a list with only a single element (`default`) for most stock apps, then shouldn't it _exclude_ the 's'? Either way you go, it feels like they should at least match, but we can't change `configTransport` now, can we?

Again, I think no good answer is possible, but thinking about "transport" as a verb instead of a noun helps. 

* `configTransport()`: "Following established convention, take the config(s) for the things doing the transporting."
* `configuredTransport()`: "Following established convention, give me the name(s) of the things doing the transporting."

¯\\_(ツ)_/¯